keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -597,13 +608,19 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+This is an opt-in feature, and it does not change any default behavior. If there is any bug in this feature, image pulls might fail.
+No running workloads will be imapcted.
+
```

Comment:
If I understand correctly, the user may bypass the limit when the node manager is constantly restarting kubelet.

The cluster/Node manager may restart the kubelet when changing the kubelet configuration to do rollout/rollback.

Not sure if we should mention it.

Reply:
I added a paragraph here mentioning that restarting kubelet might allow more image pulls than the limit